src/app/config.json

```diff
@@ -5,8 +5,25 @@
     "results": "result",
   },
   "syncOptions" : {
-    "sync_frequency" : 5,
-    "storage_strategy": "dom",
-    "do_console_log": false
+    "global":{
+      "sync_frequency" : 1,
+      "storage_strategy": "dom",
+      "do_console_log": false
+    },
+    "workorders":{
+      "sync_frequency" : 2,
```

Comment:
sorry for that forgot to change it back, had it as different times to better distinguish between them in the logs.